mirroroperator/registrymirror.py

```diff
@@ -417,7 +441,7 @@ def generate_stateful_set(self, stateful_set):
                                         mount_path="/var/lib/registry"
                                     ),
                                     client.V1VolumeMount(
```

Comment:
This needs to only be included if self.ca_certificate_bundle is set, or it's not going to work when it's empty.